modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -133,27 +152,32 @@ def add_to_apply_calls(self, func, *args, **kwargs):
 
     def drain_call_queue(self):
         """Execute all operations stored in the call queue on the object wrapped by this partition."""
+        logger = get_logger()
+        logger.debug(f"ENTER::Partition.drain_call_queue::{self._identity}")
         if len(self.call_queue) == 0:
             return
         oid = self.oid
         call_queue = self.call_queue
         if len(call_queue) > 1:
+            logger.debug(f"SUBMIT::_apply_list_of_funcs::{self._identity}")
```

Comment:
nit: on line 116 of `apply()`, the line `logger.debug(f"SUBMIT::_apply_list_of_funcs::{self._identity}")` is called after `_apply_list_of_funcs`. Is there a reason for this, or should the order be changed to be consistent?